plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestDefaultQueryBuilder.java

```diff
@@ -658,6 +746,13 @@ private static long toPrestoTimestamp(int year, int month, int day, int hour, in
         return sqlTimestampOf(3, year, month, day, hour, minute, second, 0).getMillis() * MICROSECONDS_PER_MILLISECOND;
     }
 
+    private static QueryParameter[] queryParameters(JdbcColumnHandle handle, long... values)
```

Comment:
parameter has nothing to do with column handle
pass JdbcTypeHandle to this methid